packages/react/src/auto/polaris/PolarisAutoForm.tsx

```diff
@@ -68,7 +68,7 @@ export const PolarisAutoForm = <
 
   const formContent = props.children ?? (
     <>
-      <PolarisSubmitResultBanner />
+      {!props.onSuccess && <PolarisSubmitResultBanner />}
```

Comment:
Perhaps this could be a prop into the banner as something like `showSuccessBanner` so that the failure banner can still be shown

Reply:
It might be nice to split the ResultsBanner component into two different components, one for success and one for failure.